source/computer/operating-system.rst

```diff
@@ -1,2 +1,50 @@
-操作系统
-========
+选择操作系统
+============
+
+个人电脑有三种常用的操作系统：Microsoft Windows、macOS 和 Linux。
+从市场占有率上看，Microsoft Windows 用户数目远远多于 macOS 和 Linux 用户。
+然而，大多数地震学科研工作者都选择 Linux 或 macOS 作为日常研究的主要操作系统。
+主要原因有如下几点：
+
+1. Linux 和 macOS 上可以更容易的获取和使用编程所需的开发工具
+2. 大多数地震学相关的程序/代码都是在 Linux 上编写的，且仅在 Linux 和 macOS 下可以使用
+3. Linux 和 macOS 自带了可以对数据做基本处理的命令行工具
+
+对于初学者，我们\ **建议使用 Linux 或 macOS 系统**\ ，除非你非常确定自己的科研
+工作并不需要用到别人提供的代码，或者你有足够的知识和能力在 Windows 系统下
+编译和使用别人的代码。
+
+macOS
+-----
+
+Mac 笔记本或台式机（如 MacBook Air、MacBook Pro 或 iMac）自带的 macOS 系统
+可以满足科研工作的大部分需求。对于拥有 macOS 系统的用户，建议直接使用 macOS 系统。
+可以参考《\ :doc:`/computer/macos-setup`\ 》对 macOS 系统进行进一步配置，以满足
+科研工作的需求。
+
+Linux
+-----
+
+Linux 有很多不同的发行版，所有 Linux 发行版都可以免费获取与安装。
+比较流行的发行版有
+`Ubuntu <https://ubuntu.com/>`__\ 、\
+`Debian <https://www.debian.org/>`__\ 、\
+`Fedora <https://getfedora.org/>`__\ 、\
+`CentOS <https://www.centos.org/>`__ 等。
+
+读者可以根据自己的喜好自行选择合适的 Linux 发行版。
+对于不了解 Linux 的用户，我们推荐安装 XXXX。读者可以进一步参考
```

Comment:
It's still unclear to me what the best Linux distro is for newbies.

Reply:
CentOS is dead. 
is Federo a small one? How about Ubuntu? which one has the largest user number?

Reply:
https://www.tecmint.com/top-most-popular-linux-distributions/

Maybe we can use Ubuntun. I think I will.use it.when I go back to China

Reply:
I believe ubuntu is the most popular one, but sometimes the packages are too old. Debian and Fedora update more frequently than ubuntu.

Reply:
In this way, I may choose Fedora.because I am uses to centos

Reply:
I think i will use fedora, too. But I'm not sure it's suitable for newbies, as each fedora version is maitained for 1 year.

Reply:
All the three are popular, we can just choose the one we may use in the future. we can write a manual about it just like your centos.

Reply:
I believe everyone will have a chance or have to choose which is best for them when they get used to linux

Reply:
Yes, we have to  make a choice, but no hurry.